### PR TITLE
feat: add click to pay in payment settings

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Added Click to Pay enable via Dashboard.

Uploading Screen Recording 2024-12-18 at 12.06.14 PM.mov…

## How did you test it?

Via saving and checking via request and response via Network tab. 

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [x] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by kanikabansal-juspay — DISMISSED** [review]

**Review by kanikabansal-juspay — DISMISSED** [review]

File: hyperswitch
Comment (JeevaRamu0104) [inline hyperswitch:1]: why this file has been added

Comment (PritishBudhiraja) [reply hyperswitch:1]: This I haven't added it came from some commit I guess.

Comment (JeevaRamu0104) [reply hyperswitch:1]: This not present in the main Please delete this

Comment (PritishBudhiraja) [reply hyperswitch:1]: Done

**Review by kanikabansal-juspay — APPROVED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**